packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -223,8 +223,12 @@ export class OdspDocumentService implements IDocumentService {
      */
     public async connectToDeltaStorage(): Promise<IDocumentDeltaStorageService> {
         const urlProvider = async () => {
-            const websocketEndpoint = await this.joinSession();
-            return websocketEndpoint.deltaStorageUrl;
+            if(this.policies.storageOnly === true) {
```

Comment:
Can we make it unconditional (not using joinSession)? I.e. can we always pre-compute deltaStorageUrl without doing joinSession call?
That would be ideal, as that would
- speed up catch up process
- remove a bunch of cost from boot / catch up (as we kick fetching ops right away, before container is even returned to users.

Reply:
i'm doing that is main. i want to be conservative in the release branch
https://github.com/microsoft/FluidFramework/pull/5746

Reply:
The fact that we never refresh that value through lifetime of the session (unlike what we do for detlta stream, which is called any time we lose connectivity) suggests that this URL can't change, and thus can be precomputed.